apps/test/unit/studio/itemTest.js

```diff
@@ -11,32 +16,79 @@ import {
   BEHAVIOR_GRID_ALIGNED
 } from '@cdo/apps/studio/constants';
 
-import Item from '@cdo/apps/studio/Item';
-
 describe('item', () => {
   before(() => {
     Studio.trackedBehavior = {};
     Studio.trackedBehavior.createdItems = {};
+    Studio.SQUARE_SIZE = 50;
   });
 
   let item;
 
   describe('update', () => {
+    describe('if destination is not yet set', () => {
+      before(() => {
+        // the destination-setting logic uses _.shuffle to semi-randomize the
+        // set of possible destinations. Stub that with something deterministic
+        // to make testing easier
+        sinon.stub(_, 'shuffle').callsFake(ar => ar);
```

Comment:
I don't see a matching `restore()` call and you're not using a sandbox that I can see.